.github/workflows/release.yaml

```
@@ -0,0 +1,57 @@
+name: Release Artifacts
+on:
+  release:
+    types: [published]
+
+jobs:
+  artifacts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)